types/brand.ts

```diff
@@ -0,0 +1,4 @@
+export enum Brand {
```

Comment:
for the sake of simplifying the example, I would try rolling without the complexity of different themes

Reply:
We select the autoscout theme and do not offer any switcher. We need `Brand` enum to be able to select the theme in the theme provider

Reply:
gotcha!